aws/devnet/variables.tf

```
@@ -0,0 +1,122 @@
+variable "farmer_reward_address" {
+  description = "Farmer's reward address"
+  type        = string
+}
+
+variable "domain_id" {
+  description = "Domain ID"
+  type        = list(number)
+  default     = [1, 2, 3]
+}
+
+variable "domain_labels" {
+  description = "Tag of the domain to run"
+  type        = list(string)
+  default     = ["evm", "payments"]
+}
+
+
+variable "bootstrap_node_ips" {
+  description = "IP of boostrap node"
+  type        = list(string)
+  default     = [""]
```

Do you need this default value? It makes no sense as an IP address, it'd better fail and request user to provide an IP instead of silently using invalid value.

---

These will be removed since we are strictly using AWS only, and have no need for these.